src/main/java/Bobby.java

```diff
@@ -0,0 +1,90 @@
+import java.util.Scanner;
+public class Bobby {
+    public static void main(String[] args) {
+        int counter = 0;
+        boolean isExit = false;
+        Task[] tasks = new Task[100];
+        Scanner in = new Scanner(System.in);
+        System.out.println("Hello I'm Bobby\n" + "What can I do for you?");
+
+        while (!isExit) {
+            String input = in.nextLine();
+            String command;
+            String description;
+            String by;
+            String from;
+            if (input.indexOf(' ') > 0) {
+                command = input.substring(0, input.indexOf(' '));
+            } else {
+                command = input;
+            }
+            int entry;
+            switch (command) {
+            case "bye":
+                System.out.println("See you again soon!");
+                isExit = true;
+                break;
+            case "mark":
+                entry = Integer.parseInt(input.substring(5));
+                if (entry > 0 && entry <= counter) {
+                    tasks[entry - 1].setDone(true);
+                    System.out.println("Marked as done");
+                    System.out.println(entry + "." + tasks[entry - 1]);
+                }
+                break;
+            case "unmark":
+                entry = Integer.parseInt(input.substring(7));
+                if (entry > 0 && entry <= counter) {
+                    tasks[entry - 1].setDone(false);
+                    System.out.println("Unmarked");
+                    System.out.println(entry + "." + tasks[entry - 1]);
+                }
+                break;
+            case "list":
+                for (int i = 0; i < counter; i += 1) {
+                    System.out.println((i + 1) + "." + tasks[i]);
+                }
+                break;
+            case "todo":
+                if (input.length() < 5 || input.substring(5).trim().isEmpty()) {
+                    System.out.println("Please enter a valid task.");
+                    break;
+                }
+                description = input.substring(5);
+                tasks[counter] = new Todo(description);
+                System.out.println("Okay, added:\n" + tasks[counter]);
+                counter += 1;
+                System.out.println("Now you have " + counter + " task(s) in the list.");
+                break;
```

Comment:
Maybe can consider putting if statements in a separate function -- case blocks are very long and confusing